config.js

```diff
@@ -3,11 +3,10 @@ export const API_URL =
     ? 'https://api.core.ac.uk/internal'
     : 'https://api.dev.core.ac.uk/internal'
 
-const ORIGIN = encodeURIComponent(
-  typeof window == 'object' ? window.location.origin : ''
-)
-
-export const LOGIN_URL =
-  process.env.NODE_ENV === 'production'
-    ? '/login.html'
-    : new URL(`/test/login?continue=${ORIGIN}`, API_URL).toString()
+export const getLoginPage = (fullPath = false) =>
+  new URL(
+    `/test/login?continue=${
+      fullPath ? window.location.href : window.location.origin
+    }`,
+    process.env.NODE_ENV === 'production' ? undefined : API_URL
+  ).toString()
```

Comment:
It's better to either accept full path or nothing but never boolean.

```suggestion
export const getLoginPage = (path = ORIGIN_URL) =>
  new URL(`/test/login?continue=${encodeURIComponent(path)}`).toString()
```

Reply:
https://github.com/oacore/dashboard/pull/179/commits/c6e774c7222549a9b07c48310b57a38513c40c89